runtime/gc_modron_startup/mmparseXXgc.cpp

```diff
@@ -167,10 +167,12 @@ gcParseXXgcArguments(J9JavaVM *vm, char *optArg)
 #if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
 		if (try_scan(&scan_start, "enableDoubleMapping")) {
 			extensions->indexableObjectModel.setEnableDoubleMapping(true);
+			extensions->indexableObjectModel.setEnableDoubleMappingStatus(true);
```

Comment:
we should be setting only control flag here, not status. status is set always to false by constructor of indexableObjectModel